docs/docsite/rst/galaxy/user_guide.rst

```diff
@@ -272,6 +272,9 @@ Use the following example as a guide for specifying roles in *requirements.yml*:
 
     # from locally cloned git repository (file:// requires full paths)
     - src: file:///home/bennojoy/nginx
+    
+    # from locally cloned repository
```

Comment:
This wording might be confusing since we have particular syntax for repos. The example above was recently changed https://github.com/ansible/ansible/commit/776274143fc255d24c0fba8e6b37aa38041fe471.

```yaml
# from local path
```
is maybe clearer